### DISCUSSION TITLE
GIRIER Tuya ZigBee 3.0 Light Switch Module

### DISCUSSION THREAD

**Discussion by smartroad** [Koenkk/zigbee2mqtt#19210] [Devices]
Hi, I have checked the data base for these: https://www.aliexpress.com/item/1005003417422886.html light switch modules, but it says it only supports the 1 gang. If i were to get the 2 or 3 gang is it possible it would work?

**Comment by francisp2** [comment]
I have the 1 gang and the 4 gang, and both work.

**Reply by smartroad** [reply]
Awesome thank you :)
